pkg/events/v1alpha1/events.go

```diff
@@ -23,8 +23,12 @@ const (
 	GovernorGroupsEventSubject = "groups"
 	// GovernorMembersEventSubject is the subject name for members events (minus the subject prefix)
 	GovernorMembersEventSubject = "members"
-	// GovernorMemberRequestsEventSubject is the subject name for member request events (minus the subject prefix)
-	GovernorMemberRequestsEventSubject = "members.requests"
+	// GovernorMemberRequestsNewEventSubject is the subject name for member request events (minus the subject prefix)
+	GovernorMemberRequestsNewEventSubject = "members.requests.new"
+	// GovernorMemberRequestsElevateEventSubject is the subject name for admin elevation request events (minus the subject prefix)
+	GovernorMemberRequestsElevateEventSubject = "members.requests.elevate"
+	// GovernorMemberRequestsAllEventSubject is the wildcard subject name for all group request events (minus the subject prefix)
+	GovernorMemberRequestsAllEventSubject = "members.requests.*"
```

Comment:
I'm not sure how I feel about this block of changes - the problem is that requests are no longer deterministic bc an UPDATE request for admin has the same payload as an UPDATE request for membership? 

I definitely dont think we should have `GovernorMemberRequestsAllEventSubject = "members.requests.*"` - imo, it should still be:
```
GovernorMemberRequestsEventSubject = "members.requests"
```

Reply:
Maybe there should be a `PROMOTE` and `DEMOTE` action?

Reply:
That is the problem, yes. I did it this way so that we could have all four existing event actions (create, approve, deny, revoke) emitted for both request types, but then I didn't immediately need that in the notifications addon... so I could do something else here if desired. And can definitely make that specific change. Not sure that two new actions address the same thing though.